css/index.css

```diff
@@ -104,14 +123,42 @@ p {
 .middle-content .boxes .box {
     width: 12.5%;
     height: 100px;
-    background: black;
     margin: 20px 2.5%;
     color: white;
     display: flex;
     align-items: center;
     justify-content: center;
 }
-
+#one {
+    background: teal;
+}
+#two {
+    background: gold;
+}
+#three {
+    background: cadetblue;
+}
+#four {
+    background: coral;
+}
+#five {
+    background: crimson;
+}
+#six {
```

Comment:
Great job here! This means you used class name for all of the boxes and a different id for each box! This is best practice!